mobile/src/components/chat/components/conversations-list/components/conversation/helpers/is-today-message.helper.ts

```diff
@@ -0,0 +1,12 @@
+const isTodayMessage = (messageDate: string): boolean => {
```

Comment:
Remove this helper

Reply:
we will left only one date format?

Reply:
I used this helper to show two different date formats. as on the layout: HH:mm for messages that were at current day and 'distance' for other. 
Should I left only 'distance'?